Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -1,6 +1,7 @@
 import UIKit
 @_implementationOnly import MapboxCommon_Private
 @_implementationOnly import MapboxCoreMaps_Private
+import SwiftUI
```

Comment:
Why do we need `SwiftUI` here?

Reply:
We don't. Probably an accidental auto-import.

Reply:
True this, gotta be more careful with Xcode auto-importing things.